.github/workflows/main.yml

```diff
@@ -7,7 +7,75 @@ on:
     - cron:  '0 22 * * *'
 
 jobs:
-  Linux-build-and-test:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          lfs: false
+      - name: Build and lint on Archlinux
+        run: | 
+          docker run \
+              --volume $PWD:/host \
+              --workdir /host/continuous-integration/linux \
+              --env "PYTHONDONTWRITEBYTECODE=1" \
+              archlinux/base \
+              bash -c "./setup.sh && ./build.sh && ./lint.sh"
+
+  Create-source-dist:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          lfs: false
+      - name: Create source distribution
+        run: | 
+          docker run \
+              --volume $PWD:/host \
+              --workdir /host/continuous-integration/linux \
+              --env "PYTHONDONTWRITEBYTECODE=1" \
+              ubuntu:20.04 \
+              bash -c "./setup.sh && ./create-sdist.sh"
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: sdist
+          path: dist/zivid*.tar.gz
+          retention-days: 1
```

Comment:
Is it possible to specify hours instead of days? 1 hour should be enough

Reply:
I was looking for something like that, but I could only find the argument "retention-days", no "retention-hours". 
I think 1 day should be fine though, the artifacts are like 1MB.

Reply:
Ok, sounds good